File: Java/program12.java

class DisplayClass
{
    public void Display()
    {
        int iCnt = 0;
        for(iCnt = 1; iCnt <= 4; iCnt++)
        {
              System.out.println(iCnt);  
        }
     


    }
}

class program12
{
    public static void main(String A[])
    {
        DisplayClass dobj = new DisplayClass();
        dobj.Display();

        
    }
}

File: C/program12.c
///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.c
// Description :  Display Hello on output screen 
// Autor       :  Ankita Anil Patil
// Date        :  05/05/2025
///////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

//Iteration

void Display()
{
    int iCnt=0;

    for(iCnt = 1; iCnt <= 4; iCnt++ )
    {
        printf("%d\n",iCnt);
    }
}

int main()
{
    Display();

    return 0;
}